### ISSUE TITLE
Correct way to list the servers associated with volumes

### ISSUE THREAD

**Issue by rgbkrk** [jrperritt/rack#155]
To try to find out which servers I had associated with loose volumes, I ran:

```
rack block-storage volume list --json | json -a Name | cut -d" " -f4 | rack servers instance list --stdin id
```

The type of questions I'm really asking Rackspace as a user are:

* What volumes are attached to which hosts?
* Which volumes were attached to hosts that I no longer care about?
* Which volumes are in use?

**Comment by kenperkins** [comment]
I'm a bit unsure of what you're asking in this issue? Better documentation? Code changes? both?

**Comment by rgbkrk** [comment]
Yes, I'm painting a user story for commands I'd want for volumes which would lend towards both documentation (and possibly code changes to accommodate the usage pattern).

**Comment by jnoller** [comment]
@rgbkrk Suggestions on improving the volume list / volume command with higher order functionality (volumes attached to hosts) or...?

**Comment by kenperkins** [comment]
We already support status queries on block storage:

```
$ rack block-storage volume list --status attached
```

We have some work to do for volumes attached to specific hosts, or sets of hosts. Do orphaned volumens have a distinct status?

**Comment by kenperkins** [comment]
@rgbkrk can you please be more specific on:

> Which volumes were attached to hosts that I no longer care about?

**Comment by rgbkrk** [comment]
Functionally, "Which volumes were attached to hosts that I no longer care about?" comes out to "Which volumes are not attached but were attached to a virtual host at some point?" followed by figuring out which to delete. Think the semantics for this look like

```
$ rack block-storage volume list --status=available | grep "System disk"
0028150d-3e12-4f30-bc57-b4a40f89125e	System disk for 95e2f79b-d5c9-4f22-8049-c81205eba433	50	available			SSD				[]		<nil>
```

However, I'm guessing if a server or volume errors we see a different set of states here. The user would have to check the attachments field to check if its `[]`.

**Comment by kenperkins** [comment]
Once it's disconnected, there's history to tell you that it was previously connected. @rgbkrk Is there anything left to do here? @jrperritt mentions that `volume-attachments` have been implemented, which were the other two scenarios here.

**Comment by rgbkrk** [comment]
No action at this time.
